spec/javascripts/components/intervention-spec.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env jasmine */
+/* global GOVUK */
+
+describe('The intervention component', function () {
+  const FIXTURE = `
```

Comment:
We can't use `const` in our public code because it's not supported by IE10 and not fully supported by IE11. That doesn't matter so much here because the tests aren't part of the public code, however, the tests still need to pass in IE10.

Reply:
Fixed.
How do we check that the tests pass in IE10 though?

Reply:
You can run the jasmine tests through a local browser, using `bundle exec rake app:jasmine` and going to localhost:8888 in IE10 in browserstack. Ideally we'd have that as part of the build pipeline somehow, still working on that.